client/my-sites/themes/theme-showcase.jsx

```diff
@@ -689,6 +706,7 @@ const mapStateToProps = ( state, { siteId, filter } ) => {
 			isSiteOnECommerceTrial( state, siteId ) || isSiteOnWooExpress( state, siteId ),
 		isSearchV2: ! isUserLoggedIn( state ) && config.isEnabled( 'themes/text-search-lots' ),
 		lastNonEditorRoute: getLastNonEditorRoute( state ),
+		themeTiers: getThemeTiers( state ),
```

Comment:
I'm undecided whether retrieving the ThemeTiers from state (obtained from the API), or using the `THEME_TIERS` constant directly.

Ideally, the two should always be (manually) aligned, but if a tier were to be removed in the back end, this would ensure that the dropdown was only displaying the existing, correct tiers.

Reply:
Having a single source of truth is always the better approach IMHO. If we ever need to change the tiers, and need to change them in more than one place, it would be a smell that our code didn't adhere to the SRP.

Reply:
We can't avoid defining them in two different places, unless we change the `:tier` routing fragment, which is totally not something I'm keen to include in this project. 😄 

In all other cases, though, the source of truth is the API. So I'll keep the `getThemeTiers` selector her. 👍